aDzbanuszek/people/Teacher.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+import os
+from aDzbanuszek.people.Person import Person
+
+
+class Teacher(Person):
+
+    def __init__(self, name, second_name, born_data, vacationDay, engagement):
+        super(Teacher, self).__init__(name, second_name, born_data)
+        self.vacationDay = int(vacationDay)
+        self.engagement = float(engagement)
+        self.vacationDayMax = 26
+
+    def takeVacation(self, delta):
+        if self.vacationDay == 0:
+            print('Teacher ' + Teacher.getName(self) + Teacher.getSecond_name(self) + ' do not have vacation day left')
+            return self.vacationDay == 0
+        if self.vacationDay < delta:
+            print(
+                'Teacher ' + Teacher.getName(self) + Teacher.getSecond_name(self) + ' do not have enough days')
+            return self.vacationDay == self.vacationDay
```

Comment:
nie ma potrzeby robic w tej funkcji jakiegokolwiek returna. Jej celem jest ustawienie liczby dni urlopu. Zeby sprawdzic ile ich mamy mozemy uzyc funkcji checkVacationLeft :) To samy z powyzszym returnem